arrays/arrays.c

```diff
@@ -21,11 +21,13 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
-
+  Array *Array = malloc(sizeof(struct Array));  
   // Set initial values for capacity and count
-
+  Array->capacity = capacity;
+  Array->count = -1;
   // Allocate memory for elements
-
+  Array->elements = malloc(capacity);
+  return Array;
 }
 
```

Comment:
Great job so far Ezra, I am curious as to how you came up with line 24.

Reply:
That line's based of code from the example Sean did in class. Does it look off or something?

Reply:
Nope looked it up , all good!